libr/include/r_sign.h

```diff
@@ -43,12 +51,14 @@ typedef struct r_sign_item_t {
 	RSignGraph *graph;
 	ut64 offset;
 	RList *refs;
+	RSignHash *hash;
```

Comment:
why not just have char *hash here? keeping a struct for just one field is unnecessary